CMakeLists.txt

```diff
@@ -0,0 +1,65 @@
+cmake_minimum_required(VERSION 3.0)
+
+# Compiler requirements
+set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+set(CMAKE_CXX_EXTENSIONS OFF)
+
+project(tinyply LANGUAGES CXX)
+set(PROJECT_VERSION 0.1)
+
+include_directories("${CMAKE_SOURCE_DIR}/source")
+
+# Library
+add_library(tinyply source/tinyply.cpp)
```

Comment:
It may be easier to, instead of `include_directories`, just add the header to the `add_library` call.  It shouldn't make a huge difference either way, the primary reason for this comment is to enable shared or static libraries.  It would look something like this:

```cmake
# Default to shared library, allow bypass for static if desired
option(TINYPLY_BUILD_SHARED "Build as a shared library?" ON)
if (TINYPLY_BUILD_SHARED)
  set(TINYPLY_LIBRARY_TYPE "SHARED")
else()
  set(TINYPLY_LIBRARY_TYPE "STATIC")
endif()

# The library consists of two files
add_library(
    tinyply
    ${TINYPLY_LIBRARY_TYPE}
    "${CMAKE_CURRENT_SOURCE_DIR}/source/tinyply.h"
    "${CMAKE_CURRENT_SOURCE_DIR}/source/tinyply.cpp"
)
```

Reply:
Good point, I'll add that.